src/workloads/query/TimeseriesTSBSQuery.yml

```diff
@@ -0,0 +1,1188 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This workload runs queries from the tsbs_query benchmark in Genny. It allows us to cross-check the results of the TSBS 
+  benchmarks, collect additional measurements, profile individual queries, and extend some of the queries as needed.
+  The data is preloaded by the dsi configuration. See 'configurations/test_control/test_control.tsbs_query.yml' for all 
+  the details. There are 20736000 documents in the collection.
+  There are some differences between the queries in this workload and tsbs, but we do not expect these differences to significantly
+  affect runtime:
+  1. All TSBS queries have a filter on "measurement": "cpu", which we remove since it is true for all measurements.
+  2. TSBS will generate a random start time between the start/end time of the workload for every query type. We hard code
+  our time values, since genny does not currently support date arithmetic, nor storing a fixed value from randomized generation.
+
+Keywords:
+  - timeseries
+  - aggregate
+  - group
+  - sort
+
+GlobalDefaults:
+  Database: &database cpu
+  Collection: &collection point_data
+  maxPhases: &maxPhases 20
+  Repeat: &repeat 1000 # Matches the number of queries run in 'configurations/test_control/test_control.tsbs_query.yml'.
+  # TSBS constructs a time windows of various sizes (1 hour, 8 hours, 12 hours) from a uniformly random start-time within
+  # this time-period. Ideally, we would do this as well, but currently genny doesn't support storing randomized date values as
+  # variables, so we will fix a starting value and increment it each time the query runs.
+  # StartDate: &StartDate "2016-01-04T21:49:27Z"
+  Hour1: &Hour1 "2016-01-04T22:49:27Z"
+  Hour8: &Hour8 "2016-01-05T05:49:27Z"
+  Hour12: &Hour12 "2016-01-05T09:49:27Z"
+  CloseToEndTime: &CloseToEndTime "2016-01-20T09:49:27Z"
+  StartDate: &StartDate "2016-01-04T21:49:27Z"
+
+# Create variables to mimic the randomness of tsbs_generate_queries.
+# Generates a single string like "host_42", with random integer in range [0,99].
+GenHostNames:
+  &GenHostNames {
+    ^FormatString:
+      {
+        "format": "host_%d",
+        "withArgs": [{ ^RandomInt: { min: 0, max: 99 } }],
+      },
+  }
+
+# Generates an array of strings like "host_42", with random integer in range [0,99].
+HostName1:
+  &HostName1 { ^Array: { of: *GenHostNames, number: 1, distinct: true } }
+HostName8:
+  &HostName8 { ^Array: { of: *GenHostNames, number: 8, distinct: true } }
+HostNameAll:
+  &HostNameAll { ^Array: { of: *GenHostNames, number: 100, distinct: true } }
+
+Actors:
+  # The data is already loaded into the collection from the dsi configuration.
+  - Name: Quiesce
+    Type: QuiesceActor
+    Threads: 1
```

Comment:
TSBS does either 4 or 8 threads. I don't think we need to run both configs, but let's do one of them.